Coop_Listing_Site/Coop_Listing_Site/Views/Department/Edit.cshtml

```diff
@@ -28,19 +28,112 @@
         </div>
 
         <div class="form-group">
-            @Html.LabelFor(model => model.Majors, htmlAttributes: new { @class = "control-label col-md-2" })
+            @Html.Label("Majors Not In A Department", htmlAttributes: new { @class = "control-label col-md-2" })
             <div class="col-md-10">
-                @Html.ListBox("MajorIDs", (MultiSelectList)ViewBag.Majors)
-                @Html.ValidationMessageFor(model => model.Majors, "", new { @class = "text-danger" })
+                @Html.DropDownList("Majors", "Select One To Add")
+                <button style="margin-left: 10px" type="button" class="btn" onclick="addSelectedMajor();">Add</button>
+            </div>
+        </div>
+
+        <div class="form-group">
+            @Html.Label("Majors In Department", htmlAttributes: new { @class = "control-label col-md-2" })
+            <div class="col-md-10">
+                <ol style="list-style-type: none; padding-left: 0px;">
+                    @if (Model.Majors.Count > 0)
+                    {
+                        foreach (var major in Model.Majors)
+                        {
+                            <div id="@major.MajorID">
+                                <button type="button" onclick="removeMajor(@major.MajorID);">X</button> <li style="display:inline-block;" value="@major.MajorID">@major.MajorName</li>
+                                <br />
+                            </div>
+                        }
+                    }
+                </ol>
+
             </div>
         </div>
 
         <div class="form-group">
             <div class="col-md-offset-2 col-md-10">
                 <input type="submit" value="Save" class="btn btn-primary" />
-                @Html.ActionLink("Back to List", "", "Department", new { id = Model.DepartmentID }, new { @class = "btn btn-primary" })
+                @Html.ActionLink("Back to List", "", "Department", null, new { @class = "btn btn-primary" })
             </div>
         </div>
     </div>
 }
 
+<script type="text/javascript">
```

Comment:
@Kethsar I'm stealing this method for my recipe-builder app.  This is exactly what I've been needing.